types/env.d.ts

```diff
@@ -1,8 +1,11 @@
 /// <reference types="vite/client" />
 
 interface ImportMetaEnv {
-  readonly VITE_APP_TITLE: string;
+  readonly APP_TITLE: string;
   readonly APP_GITHUB_PUBLIC_BASE_URL: string;
+  readonly APP_CONFIG_DEFAULT_SOURCE: string;
+  readonly APP_CONFIG_ALLOWED_SOURCES: string | undefined;
+  readonly APP_UNSAFE_ALLOW_DYNAMIC_CONFIG: boolean | undefined;
```

Comment:
Env properties are always typed `string | undefined`

```suggestion
  readonly APP_UNSAFE_ALLOW_DYNAMIC_CONFIG: string | undefined;
```